### DISCUSSION TITLE
Why is there a dedicated .sha256 file for Windows dists?

### DISCUSSION THREAD

**Discussion by helpermethod** [jreleaser/jreleaser#352] [Q&A] [answered]
Accoding to the docs, there should only be a `checksums_sha256.txt` which contains the checksums for all files. But at some point an additional .sha256 file got introduced, but only for Windows dists. Is this a bug?

![image](https://user-images.githubusercontent.com/1562019/128776448-063162f5-8d3d-45be-8957-8f94f0c3e293.png)

**Comment by aalmiray** [comment, marked as answer]
No. Do you have scoop configured? If so then individual checksums for matching artifacts will be automatically uploaded. Please see https://github.com/jreleaser/jreleaser/issues/287

**Reply by helpermethod** [reply]
Yes I have! Thanks for the quick reply!
